config.json

```diff
@@ -14,6 +14,7 @@
     "TempDirectory": "./tmp/",
     "UseGeoIP": true,
     "UseRegionDescriptions": true,
+    "UseRemoveIP": false,
```

Comment:
Please apply the same/appropriate changes to `config.homodea.json` (it's the file that we use).
Or let me know if I can just copy+paste (especially regarding the 'Prune' section)

Reply:
This is false by default, so left it out initially, it's relevant now so just including it here. Not important for our stuff yet.